crates/core/component/dex/src/state_key.rs

```diff
@@ -103,52 +103,61 @@ pub(crate) mod engine {
         }
     }
 
-    /// Find assets with liquidity positions from asset `from`, ordered by price.
     pub(crate) mod routable_assets {
         use penumbra_asset::asset;
         use penumbra_num::Amount;
 
         use super::*;
 
-        /// `A || be_bytes(A_from_B) => B` this will be an ordered encoding of every asset `B` directly routable to from `A`.
-        /// `a_from_b` represents the amount of `A` that can be bought with `B`.
-        /// The prefix query includes only the `A` portion, meaning the keys will be returned in order of liquidity.
-        pub(crate) fn prefix(from: &asset::Id) -> [u8; 39] {
+        /// A prefix key that takes a start asset `A` (aka. base asset) and surface adjacent
+        /// assets `B` (aka. quote asset), in ascending order of the base liquidity available.
```

Comment:
I think this description is actually a little less clear than the preceding comments